compiler/src/diagnostics/comments.re

```diff
@@ -158,7 +158,9 @@ module type OrderedComments = {
 };
 
 module MakeOrderedComments =
-       (Raw: {let comments: list(Typedtree.comment);})
+       (Raw: {
```

Comment:
looks good, just a nit but is there a reason this changed? (same as `anf_mapper.re`)?

Reply:
Reason updated.